selfhost/lexer.jakt

```diff
@@ -350,10 +350,14 @@ struct Lexer {
 
         // Everything but the quotes
         mut builder = StringBuilder::create()
-        builder.append(.input[start + 1])
+        mut i = 1uz
+        while .input[start + i] != b'\'' {
```

Comment:
This looks unbounded. What if we're at the end of the input?

Reply:
decided against the loop because byte literals should be either 1 character, or 1 backslash and 1 character.